Sources/FontRenderer.swift

```diff
@@ -161,11 +161,7 @@ extension FontRenderer {
             tok = firstCharOfNextLine
         } while (tok < end)
 
-        // The sizing using linesCount is actually correct, but it gives a different result compared to
-        // what actually gets rendered. So for now, use this updated figure to have correct sizeThatFits etc:
```

Comment:
"compared to what actually gets rendered" – sounds like there was a difference between the sizing function and the render function. i imagine that was fixed with an update to SDL_ttf yep